parity/run.rs

```diff
@@ -575,7 +575,7 @@ pub fn execute_impl(cmd: RunCmd, can_restart: bool, logger: Arc<RotatingLogger>)
 	}
 
 	// display warning if using --no-hardcoded-sync
-	if !cmd.no_hardcoded_sync {
+	if cmd.no_hardcoded_sync {
```

Comment:
does this show this warning if you actually run --light?

Reply:
Yeah, this is in a function that sets up the full client only.

Reply:
Cool

Reply:
No, the `execute_impl` function is run only when we start a full node. This `if` is never reached on the light client.